test/specs/scenarios/crossDevice.js

```diff
@@ -47,10 +48,9 @@ export const crossDeviceScenarios = async (lang) => {
     }
 
     const waitForAlertToAppearAndSendSms = async () => {
-      driver.sleep(1000)
+      driver.wait(until.alertIsPresent())
```

Comment:
What's this?

Reply:
I guess a built in selenium-webdriver function to check if there is an alert, correct?

Reply:
Correct and it works really fast ⚡️

Reply:
[Official documentation](https://github.com/SeleniumHQ/selenium/blob/master/javascript/node/selenium-webdriver/lib/until.js#L107)